src/Updater.php

```diff
@@ -97,71 +104,59 @@ public function logger() {
   /**
    * {@inheritdoc}
    */
-  public function installModules(array $modules) {
-    $successful = TRUE;
+  public function executeUpdate($module, $update_definition_name) {
+    $this->successfulExecute = TRUE;
```

Comment:
I don't think we should set it to TRUE at the beginning - nothing has been successful yet

Reply:
if anything this should be FALSE. Or we should rename it and swap the meaning to be errored